### ISSUE TITLE
请问训练到25k的时候注意力线还是没有出来，并且文件多出了一个出来这个是正常的吗？

### ISSUE THREAD

**Issue by yemaohaker** [babysor/MockingBird#234]
请问训练到25k的时候注意力线还是没有出来，并且文件多出了一个出来这个是正常的吗？

![微信图片_20211125164148](https://user-images.githubusercontent.com/54471945/143409803-3bcefd96-9709-4a3a-9523-1c9ee0f4560c.png)

![微信图片_20211125164245](https://user-images.githubusercontent.com/54471945/143409813-02495fcd-131f-4f93-a852-c0347a6440aa.png)

![attention_step_27000_sample_1](https://user-images.githubusercontent.com/54471945/143409827-920326c8-98f6-425b-8b77-0765c53fa6c8.png)

![微信图片_20211125164409](https://user-images.githubusercontent.com/54471945/143409849-f33954cf-8842-422f-9364-52b02935c21e.png)

这样的情况是正确的还是我操作错了

cpu是i59400F，显卡是1505ti，配置GPU是的数值只能配置到6，GPU的占用率已经在80-100%跳动了

Tacotron Training
    tts_schedule = [(2,  1e-3,  10_000,  6),   # Progressive training schedule
                    (2,  5e-4,  15_000,  6),   # (r, lr, step, batch_size)
                    (2,  2e-4,  20_000,  6),   # (r, lr, step, batch_size)
                    (2,  1e-4,  30_000,  6),   #
                    (2,  5e-5,  40_000,  6),   #
                    (2,  1e-5,  60_000,  6),   #
                    (2,  5e-6, 160_000,  6),   # r = reduction factor (# of mel frames
                    (2,  3e-6, 320_000,  6),   #     synthesized for each decoder iteration)
                    (2,  1e-6, 640_000,  6)],  # lr = learning rate

还请大佬给指点指点，谢谢！

**Comment by yemaohaker** [comment]
还有想问下有没有交流群可以拉下，学习学习

**Comment by ycMia** [comment]
_25k ，_50k 等文件是你在步数达到该步后生成的模型文件(相当于游戏存档了啊哈哈)

另外...我的注意力线是在步数达到72k之后才出现的...届时的loss 是 0.61, 还达不到标准
1505ti 是指1050ti的话，训练速度还要更慢一些✓ 请耐心等待

**Comment by ycMia** [comment]
>吐槽一句，我的环境跑了两个整天(48h)了都还是有明显的低频电流音...,Loss也是维持在0.6就没变过了...

**Comment by yemaohaker** [comment]
@ #吐槽一句，我的环境跑了两个整天（48h）了都还是有明显的低频电流音...，Loss也是维持在0.6就没变过了...

是的，1050ti

![image](https://user-images.githubusercontent.com/54471945/143535274-0c91e078-cad8-40be-8f9b-ea71eb6bbb7c.png)

跑得都快崩溃了哈哈，85k 还是没有注意力线

![image](https://user-images.githubusercontent.com/54471945/143535327-0e0a4bb0-ceee-487b-9462-a4e0560c1ef7.png)

![image](https://user-images.githubusercontent.com/54471945/143535362-bb3aaf58-1cba-4d59-9aa2-6ac832775bb3.png)

**Comment by yellow1choco** [comment]
同1050TI 已经75k了，loss浮动在0.8-0.5之间，多数分布在0.6
![attention_step_73500_sample_1](https://user-images.githubusercontent.com/49072485/143563205-3c5eee5d-eead-4011-8da5-209ed1630c17.png)
![step-73500-mel-spectrogram_sample_1](https://user-images.githubusercontent.com/49072485/143563219-c78e02de-34fa-44c2-88df-b7efb0e08a3c.png)

**Comment by babysor** [comment]
hmm。看这个attention图貌似问题比较大，考虑基于我分享的模型上跑？

**Comment by delcompan** [comment]
在别人分享的模型基础之上继续训练，训练集要求和之前的一样吗

**Comment by babysor** [comment]
> 在别人分享的模型基础之上继续训练，训练集要求和之前的一样吗。我的显存8g，batchsize设置多少不会报错，太小了就没必要折腾了

训练集可以完全不一样，batchsize在20以内，你可以试探一下，只要不崩溃就可以调大

**Comment by Nolan-HK** [comment]
你的数据集用了多少条音源？，做下参考

**Comment by yrsn509** [comment]
我20k出现注意力线还以为是最慢的了。。

**Comment by 1239hy** [comment]
> 嗯。看这个attention图貌似问题比较大，考虑基于我分享的模型上跑？

想请教，您分享打的模型在哪里看到？我要咋操作在模型基础上继续跑？
要重新开始吗？

**Comment by pxdawn** [comment]
> > 在别人分享的模型基础之上继续训练，训练集要求和之前的一样吗。我的显存8g，batchsize设置多少不会报错，太小了就没必要折腾了
> 
> 训练集可以完全不一样，batchsize在20以内，你可以试探一下，只要不崩溃就可以调大

请问怎么调整batch_size，没有找到参数设置位置，默认好像是12？

**Comment by Nolan-HK** [comment]
可以下载别人做好的模型，名字改成和自己数据集一样就行。
